website/docs/tutorials/02-installation.md

```diff
@@ -4,45 +4,50 @@ sidebar_label: Installation
 
 # Installation
 
+Protostar is a valuable tool for developing StarkNet smart contracts, and it is easy to install on Linux and Mac systems. In this section, we will walk through the process of installing and upgrading Protostar, as well as the requirements for using the tool.
+
 ## Requirements
-Protostar requires a [git](https://git-scm.com/) executable to be specified in the `$PATH`.
+To use Protostar's dependency management commands, you must have git installed and added to your `$PATH` environment variable.
+
+The `$PATH` is an environment variable that specifies a list of directories in which the system looks for executables.
+If you do not have git in your `$PATH`, you will need to add the directory where git is installed in order to use Protostar.
 
 ## Linux and Mac
-1. Copy and run in a terminal the following command:
+1. Open a terminal and enter the following command:
 ```console
 curl -L https://raw.githubusercontent.com/software-mansion/protostar/master/install.sh | bash
 ```
-2. Restart the terminal.
-3. Run `protostar -v` to check Protostar and [cairo-lang](https://pypi.org/project/cairo-lang/) version.
+2. Close and reopen the terminal.
```

Comment:
Why do we need to restart the terminal? this shouldn't be a thing

Reply:
Cannot we add export to the script so it would work instantly?

Reply:
https://github.com/software-mansion/protostar/issues/1350